src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/COperators.cs

```diff
@@ -128,12 +128,6 @@ private static PredefinedName GetMethodName(OperatorKind op)
             return GetInfo(op).methodName;
         }
 
-        private static Name GetMethodName(NameManager namemgr, OperatorKind op)
-        {
-            Debug.Assert(HasMethodName(op));
-            return namemgr.GetPredefName(GetMethodName(op));
```

Comment:
The call to `GetPredefName` is now at [line 110](https://github.com/dotnet/corefx/pull/17605/files#diff-e0d7f2adda97e55134497c1318810263R110) where it is on the right-hand side of `HasMethodName(i) &&` so it's more blatant that what was asserted here must hold.